test/e2e/operator_test.go

```diff
@@ -250,79 +250,40 @@ func TestDNSLogging(t *testing.T) {
 	}
 	catCmd := []string{"cat", "/etc/coredns/Corefile"}
```

Comment:
*nit* I think naming this variable would help understand the logic.

```suggestion
	catCorefileCmd := []string{"cat", "/etc/coredns/Corefile"}
```